gobblin-metrics-libs/gobblin-metrics-base/src/main/java/gobblin/metrics/reporter/ConfiguredScheduledReporter.java

```diff
@@ -219,7 +233,8 @@ protected String getMetricNamePrefix(Map<String, Object> tags){
     if (metricContextName == null || (currentContextName.indexOf(metricContextName) > -1)) {
       return currentContextName;
     }
-    return JOINER.join(metricContextName, tags.get("taskId"), tags.get("forkBranchName"), tags.get("class"));
+
+    return JOINER.join(metricsPrefix, metricContextName, tags.get("taskId"), tags.get("forkBranchName"), tags.get("class"));
```

Comment:
Does `Joiner.skipNulls()` skip empty strings? If not, this might produce an undesired string when the prefix is empty.